express-api/src/services/projects/projectsServices.ts

```diff
@@ -828,7 +830,8 @@ const getProjects = async (filter: ProjectFilter) => {
       logger.error('PropertyUnion Service - Invalid Sort Key');
     }
   }
-  return await query.getMany();
+  const projects = await query.getMany();
+  return { projects, totalCount };
```

Comment:
There's an existing call that seems to work here and would do this all in one. 
```suggestion
  const [properties, totalCount] = await query.getManyAndCount();
  return { properties, totalCount };
```